Deprecate AT%XRAI functionality (#12307)
nrfconnect/sdk-nrf: samples/cellular/udp/README.rst

CONFIG_UDP_DATA_UPLOAD_FREQUENCY_SECONDS - UDP data upload frequency configuration
   This configuration option sets the frequency with which the sample transmits data to the server.

.. _CONFIG_UDP_SERVER_IP_ADDRESS:

CONFIG_UDP_SERVER_IP_ADDRESS - UDP Server IP Address configuration
   This configuration option sets the static IP address of the server.

.. _CONFIG_UDP_SERVER_PORT:

CONFIG_UDP_SERVER_PORT - UDP server port configuration
   This configuration option sets the server address port number.

.. _CONFIG_UDP_RAI_ENABLE:

CONFIG_UDP_RAI_ENABLE - RAI configuration
   This configuration option, if set, allows the sample to request RAI for transmitted messages.

.. note::

>    PSM and eDRX are configured using options for the :ref:`lte_lc_readme` library.

Reviewer: ```suggestion
   To configure PSM and eDRX, use the options from the :ref:`lte_lc_readme` library.
```
